docs/known_bugs.rst

```diff
@@ -84,6 +84,17 @@ where you do not see this error or use Jupyter Notebook instead.
 Cubeviz
 -------
 
+Spectrum does not appear when running on a Linux VM
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When running Jdaviz on a Linux virtual machine (VM), the spectrum may not appear
+in the spectrum viewer. This is a `known bug <known-gl-issue>` in an underlying
```

Comment:
Where is `known-gl-issue` defined? I don't see it in the diff here.

Reply:
Oh ffs, I had the wrong text in my clipboard when I pasted